README.md

```diff
@@ -40,5 +40,5 @@ test run, show output to console, transcoding to ogg at level 3 (default = 1)
 - note that lossy folders *must* exist before transcoding
 - when given --logfolder, operations will be logged to transcoder.log
   - watch log: tail -f <logfolder>transcoder.log   
-- for embedding album art in the transcoded files, the script expects a file cover.jpg 
-  in the lossless (flac) folder; otherwise no embedding can take place
+- for embedding album art in the transcoded files, the script expects a file *"cover.jpg"* in the lossless (flac) folder; otherwise no embedding can take place
+  - to replace an album cover, remove the transcoded album and simply replace *"cover.jpg"* in the lossless (flac) folder. (However, the cover will not be replaced in the lossless flac files)
```

Comment:
De opmerking tussen haakjes mag weg, is verwarrend

Reply:
En eigenlijk is dit een bug.....

Reply:
Het getranscodeerde album hoeft niet verwijderd te worden. Het transcoderen vindt sowieso plaats omdat de flacs nieuwer zijn. En omdat cover.jpg wordt *vervangen*  door het nieuwere exemplaar, zal het embedden dus ook met de nieuwe cover-file plaatsvinden. Maw.... deze hele opmerking kan weg

Reply:
Ok. Ik geef hier aan dat het plaatje cover.jpg vervangen wordt, dit niet in de FLAC bestanden vervangen wordt.

Reply:
Zo staat het er niet. En het is niet de taak van de transcoder om cover art in de flacs te embedden. Wel in de getranscodeerde files. Maar dat doet de transcoder dan weet prima. Zoals hierboven aangegeven